intro/src/main/java/mate/academy/intro/model/ShoppingCart.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.intro.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
+@SQLDelete(sql = "UPDATE shopping_carts SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "shoppingCart")
```

Comment:
It is better to use the `Lazy` FetchType
If you use the `EAGER` you will have bad performance
```suggestion
    @OneToMany(fetch = FetchType.Lazy, mappedBy = "shoppingCart")
```